src/m365/commands/status.ts

```diff
@@ -14,29 +14,39 @@ class StatusCommand extends Command {
   }
 
   public commandAction(logger: Logger, args: any, cb: (err?: any) => void): void {
-    if (auth.service.connected) {
```

Comment:
I tested the current setup with an expired access token. All calls are failing now with an exception:
![image](https://user-images.githubusercontent.com/18241305/180648571-5cf43787-2139-4964-ae39-a0825a4c0a14.png)

![image](https://user-images.githubusercontent.com/18241305/180648587-abc7e53c-84ea-45dc-84fe-16cc43cd1d1f.png)

I do not think that you should remove the if for auth.service.connected

Reply:
@plamber i cannot re-produce this issue with an expired access token. Did you mean an invalid 'refresh token' ? from the discussion in #3260 , we wanted to always fetch the token to see if the user is connected, instead of using the `if(auth.service.connected)` property. 
In the first screenshot, I believe, we can remove the `cb(new CommandError(rej.message));` from the updated code so that it only shows "Logged Out". 
In the second screenshot, is some other error expected ? I might be missing something here

Reply:
The problem is that I have an expired access token and that now whatever command is being executed I am getting the exception above. I am trying to reproduce it again and come back to you